build.gradle

```diff
@@ -61,6 +61,10 @@ java {
 application {
     mainClass.set('org.jabref.Launcher')
     mainModule.set('org.jabref')
+
+    applicationDefaultJvmArgs =
+        // Fix for https://github.com/JabRef/jabref/issues/11188
+        ['--add-exports=javafx.base/com.sun.javafx.event=org.jabref.merged.module']
```

Comment:
seems like we need to add all runStuff here 
'> java.lang.reflect.InaccessibleObjectException: Unable to make protected javafx.collections.ObservableList javafx.scene.Parent.getChildren() accessible: module javafx.graphics does not "opens javafx.scene" to module org.jabref.merged.module 

from https://github.com/JabRef/jabref/issues/11198